easybuild/easyconfigs/l/libplinkio/libplinkio-0.9.8-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,37 @@
+##
+# This easyconfig is contributed by DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage:     https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::     Robert Qiao <rob.qiao@flinders.edu.au>
+# License::     BSD
+#
+# Notes::
+##
+
+easyblock = 'CMakeMake'
+
+name = 'libplinkio'
+version = '0.9.8'
+
+homepage = "https://github.com/mfranberg/libplinkio"
+description = """A small C and Python library for reading PLINK genotype files."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = ['https://github.com/mfranberg/%(namelower)s/archive/refs/tags']
+sources = ['v%(version)s.tar.gz']
+checksums = ['de54dd4789b2da9b937a1f445dc6976153211dc35b4376a4cf561ad2ee861075']
+
+builddependencies = [
+    ('CMake', '3.15.3'),
+    ('binutils', '2.32'),
+]
+
+parallel = 1
+
+sanity_check_paths = {
+    'files': ['lib/%(name)s.%s' SHLIB_EXT],
```

Comment:
```suggestion
    'files': ['lib/%(name)s.%s' % SHLIB_EXT],
```

Reply:
@manifestoso I think that will fix the issue. Line numbers ignore comments at the start of an easyconfig - so in this case the error will be reported against the wrong line.